examples/feature_engineering/write_once_run_everywhere_blog_post/components/aggregations.py

```diff
@@ -26,3 +26,15 @@ def age_mean__online() -> float:
 def age_stddev__online() -> float:
     """Load the previously computed aggregation data"""
     return utils.query_scalar("age_stddev")
+
+
+@config.when(mode="streaming")
```

Comment:
Maybe have a comment that we could also use bytewax's stateful aggregator (although in this case it wouldn't make sense)